functions.js

```diff
@@ -74,11 +70,32 @@ function clickLogic (clickedCard, deck) {
             divId : clickedCard.id,
             url : clickedCard.childNodes[0].src
         }
+        turnCounter++
+        document.getElementById('turnContainer').textContent = String(turnCounter)
         if (comparison(clickedCard1.url, clickedCard2.url)) {
             clickCounter = 0
+            if(winState()) {
+                gameEnd()
+            }
         } else {
             clickCounter = 2
             cardHide(clickedCard1, clickedCard2)
         }
     }
+}
+
+function gameEnd () {
+    document.getElementById('totalTurns').textContent = String(turnCounter)
+    $('#winScreen').fadeIn(1500)
+}
+
+function winState() {
```

Comment:
this function can be achieved in a single line. Look up https://www.w3schools.com/jsref/jsref_every.asp